### ISSUE TITLE
tf undetermined

### ISSUE THREAD

**Issue by sss-source** [Luffy212/DD-GAN#5]
def MSE_LOSS(self, label, pred):
        return tf.losses.mean_squared_error(label, pred)

What does tf represent and why is it displayed as undefined?
